frontend/src/lifetime/input.ts

```diff
@@ -626,5 +628,5 @@ function isKeyPrintable(key: string): boolean {
 		"ZoomToggle",
 	];
 
-	return !allPrintableKeys.includes(key);
+	return !ALL_PRINTABLE_KEYS.includes(key);
```

Comment:
Nitpick. `ALL_PRINTABLE_KEYS` should be defined above the function since it is doing an array allocation per function call when it should just be a constant value. In addition, you could use a `Set<string>` to have O(1) lookup